README.rst

```diff
@@ -43,7 +43,8 @@ Install updated version of the following Python packages::
 Note the ``--user`` flag, which instructs Pip to not overwrite system
 files. You must add ``$HOME/.local/bin`` to your ``$PATH`` to be able to run
 the updated Pip, e.g., by adding the following line to ``$HOME/.profile``
-and launching a new login shell (e.g., logout or execute ``bash -l``)::
+and launching a new login shell (e.g., logout or execute ``bash -l`` 
+or execute ``source ~/.profile``)::
```

Comment:
I am not sure it is a good idea to recommend sourcing `.profile`. Executing some of the code in there twice in the same shell might cause problems.

If you want to add a simple instruction, tell users to execute the line.

Reply:
well, either way is fine with me. I will drop the source instruction.